.travis.yml

```
@@ -31,26 +31,21 @@ jobs:
   include:
     # Tests the lowest set of dependencies
     - php: 7.1
-      env: LOWEST SYMFONY_DEPRECATIONS_HELPER=999999
+      env: LOWEST SYMFONY_DEPRECATIONS_HELPER=weak
       install:
         - travis_retry composer update -n --prefer-lowest --prefer-stable --prefer-dist
 
-    # Test against latest Symfony 3.4 stable
-    - php: 7.3
-      env: SYMFONY_VERSION="3.4.*" SYMFONY_DEPRECATIONS_HELPER=999999
-
     # Test against latest Symfony 4.3 stable
     - php: 7.3
-      env: SYMFONY_VERSION="4.3.*"
+      env: SYMFONY_REQUIRE="4.3.*"
       install:
         - composer require --dev "symfony/messenger:4.3.*" --no-update
-        - travis_retry composer update -n --prefer-dist
+        - travis_retry composer update -n --prefer-dist --prefer-stable
 
     # Test against latest Symfony 4.4 dev
     - php: 7.3
-      env: SYMFONY_VERSION="4.4.*@dev"
+      env: SYMFONY_REQUIRE="4.4.*"
       install:
-        - composer config minimum-stability dev
         - composer require --dev "symfony/messenger:4.4.*@dev" --no-update
```

Does this still need `@dev` if we're removing it above?

```suggestion
        - composer require --dev "symfony/messenger:4.4.*" --no-update
```